docs/source/eFeatures.rst

```diff
@@ -1657,7 +1657,7 @@ The ratio between the voltage deflection and stimulus current
 
 - **Required features**: t, V, stim_start, stim_end, voltage_deflection
 - **Parameters**: stimulus_current
-- **Units**: mV/nA
+- **Units**: MΩ
```

Comment:
can greek letters be parsed by sphinx from rst? Have you tried making the docs locally to check that?

Reply:
I tried but couldn't succeed. How do you build the docs?

When I try `make doc`, it proceeds for a while but then it asks me to enter a file name. :thinking: 
```
This is pdfTeX, Version 3.141592653-2.6-1.40.22 (TeX Live 2022/dev/Debian) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
(./efeature-documentation.tex
LaTeX2e <2021-11-15> patch level 1
L3 programming layer <2022-01-21>
(/usr/share/texlive/texmf-dist/tex/latex/base/article.cls
Document Class: article 2021/10/04 v1.4n Standard LaTeX document class
(/usr/share/texlive/texmf-dist/tex/latex/base/size10.clo))
(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsmath.sty
For additional information on amsmath, use the `?' option.
(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amstext.sty
(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsgen.sty))
(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsbsy.sty)
(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsopn.sty))

! LaTeX Error: File `boxedminipage.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

Enter file name: 
```

Reply:
I usually use tox to do it, but it looks like there is no docs building in the efel tox.

Maybe take inspiration from https://github.com/BlueBrain/EModelRunner/blob/main/tox.ini#L94 to build the docs?

Reply:
I can try to write something similar to emodelrunner but I am wondering  how is the docs updated in efel in usual? I.e. how did the current docs got built?

Reply:
The issue was from my system, it was missing this Debian package: texlive-latex-extra
Omega gets rendered fine. The theme of the docs is old btw. let me see if it can be updated

![Screenshot_20230428_233459](https://user-images.githubusercontent.com/7026020/235258202-94a97c9f-f245-4762-9416-6df0574d8c17.png)

Reply:
I tried the sphinx blue brain theme, it looks like this. The feature names are not in bold however. The rst needs to be updated I think. Let's consider doing that in another PR?
![Screenshot_20230501_101259](https://user-images.githubusercontent.com/7026020/235427452-ecbc1731-1e48-4093-a21c-0fcb531191b3.png)

Reply:
sounds good!